index.html

```diff
@@ -46,8 +44,110 @@ <h2 class="job"> I’m a software developer</h2>
+            <img src="/images/Imgplaholder.svg" alt="Image placeholder">
+            <h2 class="recent-work__items"> Multi-Post Stories Gain+Glory </h2>
+            <ul class="recent-work__items-languages">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button type="button" class="see-projects">See Projects</button>
+         </article>
+         <article class="article-card">
+            <img src="/images/Imgplaholder.svg" alt="Image placeholder">
+            <h2 class="recent-work__items"> Multi-Post Stories Gain+Glory </h2>
+            <ul class="recent-work__items-languages">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button type="button" class="see-projects">See Projects</button>
+         </article>
+         <article class="article-card">
+            <img src="/images/Imgplaholder.svg" alt="Image placeholder">
+            <h2 class="recent-work__items"> Multi-Post Stories Gain+Glory </h2>
+            <ul class="recent-work__items-languages">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button type="button" class="see-projects">See Projects</button>
+         </article>
+      </section> 
+<section class="profile">
+    <div class="abouttop">
+        <img src="/images/aboutup.svg" alt="" class="aboutup">
+        <img src="/images/aboutdown.svg" alt="" class="aboutdown"> 
+    <h2 class="profile-me">About me</h2>
+    <p class="profile-intro"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+    <button type="button" class="get-resume">Get My Resume</button>
+</div>
+  <ul class="resume-items">
+    <li class="languages">
+        <img src="/images/Languages.svg" alt="Languages Image"> 
+        <h2>Languages</h2>
+        <ul class="my-languages">
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+        </ul>
+    </li>
+    <li class="languages">
+        <img src="/images/Frameworks.svg" alt="Frameworks Image">
+        <h2>Frameworks</h2>
+        <ul class="my-languages">
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+        </ul>
+    </li>
+    <li class="languages">
+        <img src="/images/Skills.svg" alt="Skills Image">
+        <h2>Skills</h2>
+        <ul class="my-languages">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
```

Comment:
- [ ] Please Fix the link to the icon and images instead of /images write./images instead

Reply:
@MeqdamAlqudah thank you for your review. I have made the changes requested but I'm not seeing how it affected the elements. Can you please explain why to use ./images instead of /images?